lib/client/api.go

```diff
@@ -312,6 +312,10 @@ type Config struct {
 	// SessionID is a session ID to use when opening a new session.
 	SessionID string
 
+	// extraEnvs contains additional environment variables that will be added
+	// to SSH session.
+	extraEnvs map[string]string
```

Comment:
It's a little strange that this is not exported while everything else is. Are callers outside of this package not supposed to be able to set this?

Reply:
Callers outside of this package should use `AddEnv()` method https://github.com/gravitational/teleport/pull/25955/files#diff-9fedd696f2d0af6a43723bf16cadcc57adc8f30ca98833baaf8b560f8f8b8f80R1669